src/csvcubed/models/sparqlresults.py

```diff
@@ -582,3 +663,21 @@ def map_table_schema_properties_result(
         table_url=str(result_dict["csvUrl"]),
     )
     return result
+
+
+def map_is_pivoted_shape_for_measures_in_data_set(
+    sparql_results: List[ResultRow],
+) -> List[IsPivotedShapeMeasureResult]:
+    """
+    Maps the sparql query result to objects of type IsPivotedMeasureResult that are then returned.
+    """
+    is_pivoted_shape_measure_results = list(
+        map(
+            lambda result: IsPivotedShapeMeasureResult(
+                measure=str(result.asdict()["measure"]),
```

Comment:
Again, can you make sure to only call `.asdict` once per result given that I think this is likely an expensive operation?